src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/databaseFeedManager.hpp

```diff
@@ -547,10 +547,16 @@ class TDatabaseFeedManager final : public Observer<nlohmann::json&>
     /**
      * @brief Get CNA/ADP name based on the package vendor.
      * @param vendor Package vendor.
+     * @param format Package format.
      * @return CNA/ADP name.
      */
-    std::string getCnaName(std::string_view vendor)
+    std::string getCnaName(std::string_view vendor, std::string_view format)
     {
+        if (!format.compare("pypi"))
```

Comment:
I'd add a comment to explain this temporary workaround.

Reply:
done